frontend/src/pages/Scans/ScansView.tsx

```diff
@@ -184,6 +204,7 @@ const ScansView: React.FC = () => {
       // For now, parse the arguments as JSON. We'll want to add a GUI for this in the future
       let body: typeof values = Object.assign({}, values);
       body.arguments = JSON.parse(values.arguments);
+      if (values.isSingleScan) body.frequency = 1;
```

Comment:
Did you mean to set `body.IsSingleScan` instead?

Reply:
Since frequency does not matter for single scans, I set frequency to an arbitrary 1, in order to prevent a status code 400 for cases when frequency is set to anything <1 by the user.